packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx

```diff
@@ -51,19 +55,27 @@ const defaultProps = {
   isAnimated: false,
   isInitiallyOpen: false,
   isTransparent: false,
+  isOpen: false,
 };
 
 class ToggleSectionHeader extends React.Component {
   constructor(props) {
     super(props);
 
     const { isInitiallyOpen = false } = this.props;
-    this.state = { isOpen: isInitiallyOpen };
+    this.state = { isOpen: isInitiallyOpen || this.props.isOpen };
 
     this.handleOnClick = this.handleOnClick.bind(this);
     this.wrapOnClick = this.wrapOnClick.bind(this);
   }
 
+  componentDidUpdate(previousProps, previousState) {
```

Comment:
I think we can catch this update earlier in the lifecycle, updating here requires two renders for each update.

https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops

Reply:
Updated ed43374